nise/report.py

```diff
@@ -75,46 +78,89 @@ def route_file(bucket_name, bucket_file_path, local_path):
                      local_path)
 
 
+def _create_month_list(start_date, end_date):
+    """Create a list of months given the date range args."""
+    months = []
+    current = start_date
+
+    if start_date.month == end_date.month:
+        # No month range
+        month = {}
+        month['name'] = calendar.month_name[start_date.month]
+        month['start'] = start_date
+        month['end'] = end_date
+        months.append(month)
+    else:
+        while current.month <= end_date.month:
```

Comment:
Why do we need an if/else here seems like the while loop could handle both cases if you changed the if/elif to just to if conditions. Am I wrong?

Reply:
Ah yeah, good catch.